### PR TITLE
Onboard designate operator

### PR DESCRIPTION
**Pull request by bshewale**

### CODE REVIEW

File: ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  from: src
+  to: openstack-operator-ci-pre-commit
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml:16]: make gotest is not covered by pre-commit, can we can as  job here?
```
- as: unit
  commands: make gotest
  container:
    from: src
```

File: ci-operator/config/openstack-k8s-operators/designate-operator/OWNERS [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - Sandeepyadav93
+  - stuggi
+  - viroel
+  - weinimo
+  - gibizer
+
+reviewers:
+  - abays
+  - beagles
+  - johnsom
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/designate-operator/OWNERS:16]: johnsom is not part of OpenShift CI org, we need to ask him to join the org.

Comment (Sandeepyadav93) [reply ci-operator/config/openstack-k8s-operators/designate-operator/OWNERS:16]: Yes, we requested him to join the Org.

File: ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml (hunk 1 above)
Comment (Sandeepyadav93) [reply ci-operator/config/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main.yaml:16]: Hey Doug,

Designate operator don't have any gotests currently, I requested bhagyashri to drop this job and we can readd once we have the gotests.

**Comment by viroel** [comment]
/pj-rehearse ack

new jobs are passing. See pre-commit logs[1]

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37982/rehearse-37982-pull-ci-openstack-k8s-operators-designate-operator-main-precommit-check/1643200797753741312/artifacts/test/build-log.txt

**Pull request merged**